src/client/.storybook/main.ts

```diff
@@ -0,0 +1,26 @@
+import path from "path"
+import { mergeConfig } from "vite"
+
+module.exports = {
+  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
+  framework: {
+    name: "@storybook/react-vite",
+    options: {},
+  },
+  features: {
+    storyStoreV7: true,
```

Comment:
Actually this can be removed. it was set to false because in SB7 it is true by default and the `storiesOf` api is not compatable with 7 and we were getting errors. 
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#70-deprecations-and-default-changes